python/dgl/frame.py

```diff
@@ -120,9 +121,14 @@ class Column(object):
         Index tensor
     """
     def __init__(self, storage, scheme=None, index=None, device=None):
-        self.storage = storage
-        self.scheme = scheme if scheme else infer_scheme(storage)
-        self.index = index
+        if isinstance(storage, tuple):
+            # handle DistTensor and node/edge ids in subgraphs
+            self.storage = storage[0]
+            self.index = storage[1]
+        else:
+            self.storage = storage
+            self.index = index
+        self.scheme = scheme if scheme else infer_scheme(self.storage)
```

Comment:
Try to avoid changing the `__init__` function as I suggested earlier.

Reply:
Removed this part and rolled back to the original implementation.